.github/ISSUE_TEMPLATE/iceberg_bug_report.yml

```diff
@@ -0,0 +1,97 @@
+---
+name: Iceberg Bug report 🐞
+description: Problems, bugs and issues with Apache Iceberg
+labels: ["kind:bug"]
```

Comment:
Should we be defaulting all new issues as `kind:bug`? I would think we should be adding that when something is determined to be a bug (or possible bug). We get a lot of issues that are seeking assistance, or hoping to implement new features etc.

Reply:
Looking at the Airflow issues, I see now how this would be a good label to use by default (given the user selected bug): https://github.com/apache/airflow/issues/new/choose